### PR TITLE
Team section photos

### PR DESCRIPTION
**Pull request by hasparus**
### Background

https://github.com/user-attachments/assets/a0ad1695-3842-4369-8ab6-1b628a3ecae1

### Description

I’ve used the GitHub photos for everybody except of Noam, as discussed.
Can grab the ones from Figma, but I’m not sure if the Hive repo should be the source of truth for those.

### CODE REVIEW

File: packages/web/docs/src/components/team-section/index.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
+import Image, { StaticImageData } from 'next/image';
+import { GlobeIcon } from '@radix-ui/react-icons';
+import { DiscordIcon, GitHubIcon, TwitterIcon } from '@theguild/components';
+import { cn } from '../../lib';
+import { ArrowIcon } from '../arrow-icon';
+import { CallToAction } from '../call-to-action';
+import { Heading } from '../heading';
+import dimaPhoto from './dima.webp';
+import noamPhoto from './noam.webp';
+import saihajPhoto from './saihaj.webp';
```
Comment (saihaj) [inline packages/web/docs/src/components/team-section/index.tsx:10]: I guess mine and Dima can be pulled from GH too?

File: packages/web/docs/src/components/team-section/index.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    'https://avatars.githubusercontent.com/kamilkisiela?v=4&s=180',
+    'https://github.com/kamilkisiela',
+  ],
+  [
+    'Laurin Quast',
+    'https://avatars.githubusercontent.com/n1ru4l?v=4&s=180',
+    'https://github.com/n1ru4l',
+  ],
+  ['Noam Malka', noamPhoto, 'https://noam-malka.com/'],
+  ['Saihajpreet Singh', saihajPhoto, 'https://github.com/saihaj'],
```
Comment (saihaj) [inline packages/web/docs/src/components/team-section/index.tsx:83]: https://avatars.githubusercontent.com/u/44710980?v=4 ?

**Review by saihaj — APPROVED** [review]

File: packages/web/docs/src/components/team-section/index.tsx (hunk 1 above)
Comment (hasparus) [reply packages/web/docs/src/components/team-section/index.tsx:10]: Theoretically, yes...
Practically, see the commit name.

https://github.com/kamilkisiela/graphql-hive/pull/5447/commits/d0f822126b749a7badc9100534245b4e8e2df81d

Comment (saihaj) [reply packages/web/docs/src/components/team-section/index.tsx:10]: ohh interesting okay!

**Pull request merged**